.github/actions/report-ci-errors/README.md

```diff
@@ -0,0 +1,26 @@
+## Pre-requisites
+
+You will need to get the slack webhook url from the secrets manager first and add it as SLACK_WEBHOOK_URL as a repository secret. 
+
+## Usage example:
+
+name: Always failing job
+on:
+  workflow_dispatch:
+
+jobs:
+  failed-run:
+    name: Failed run
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Always failing
+        run: exit 1
```

Comment:
I'd use a real use case example here like `name: Deploy`. This is how the workflow can be tested so might be a bit confusing/unhelpful.

Reply:
The code here is just for showing where the code is inserted and used rather than being a demonstration of a use case

Reply:
I've added a note to state where the action might be typically used